### PR TITLE
Add build instructions to README.md

### PR DESCRIPTION
**Pull request by jacksonargo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+
+### Build with go
+
+```
+# Clone the repo
+$ git clone https://github.com/virtual-vgo/vvgo.git
+# Build it
+$ go build -o vvgo
+# Run it
+$ ./vvgo
```
Comment (0az) [inline README.md]: As a matter of style, I'd drop the `$`: it makes single line copy/paste harder. It also allows for pasting the entire thing, which isn't a good idea, but that's on them.

Comment (0az) [reply README.md]:
```suggestion
# Clone the repo
git clone https://github.com/virtual-vgo/vvgo.git
# Build it
go build -o vvgo
# Run it
./vvgo
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+$ git clone https://github.com/virtual-vgo/vvgo.git
+# Build the docker image
+$ docker build --tag vvgo .
+# Start the container
+$ docker run -p8080:8080 --rm vvgo
+```
+
+### Build with go
+
+```
```
Comment (0az) [inline README.md]:
```suggestion
```sh
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+# Clone the repo
+$ git clone https://github.com/virtual-vgo/vvgo.git
+# Build it
+$ go build -o vvgo
+# Run it
+$ ./vvgo
+
```
Comment (0az) [inline README.md]: Okay, GH won't let me suggest the closing backticks, but they're missing.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+### Build with docker
+
+```
+# Clone the repo
+$ git clone https://github.com/virtual-vgo/vvgo.git
+# Build the docker image
+$ docker build --tag vvgo .
+# Start the container
+$ docker run -p8080:8080 --rm vvgo
+```
```
Comment (0az) [inline README.md]: Same as below.
```suggestion
```sh
# Clone the repo
git clone https://github.com/virtual-vgo/vvgo.git
# Build the docker image
docker build --tag vvgo .
# Start the container
docker run -p8080:8080 --rm vvgo
```
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
...
+# Start the container
+docker run -p8080:8080 --rm vvgo
+```
+
+### Build with go
+
+```sh
+# Clone the repo
+git clone https://github.com/virtual-vgo/vvgo.git
+# Build it
```
Comment (jacksonargo) [inline README.md]: cd vvgo

**Pull request merged**